test/powershell/Modules/Microsoft.PowerShell.Management/Remove-Item.Tests.ps1

```diff
@@ -109,6 +112,14 @@ Describe "Remove-Item" -Tags "CI" {
 	    Test-Path (Join-Path -Path $testpath -ChildPath file1.wav) | Should -BeFalse
 	    Test-Path (Join-Path -Path $testpath -ChildPath file2.wav) | Should -BeFalse
 	}
+
+        It "Should be able to remove file when path contains special char" {
+            New-Item -Path $testpath -Name $testfileSpName -ItemType File -Force
+            $testfilepathSp | Should -Exist
+
+            Remove-Item -Path $testfilepathSp
+            $testfilepathSp | Should -Not -Exist
+        }
```

Comment:
Please correct a indentation of the It block.

Reply:
I believe my indentation is correct.
What I notice is, there are 93 lines of code that is mixing tab/space for indentation.
Should I fix them here?

Reply:
Sorry, GitHub show a shift.
Closed.

Reply:
I hope we already fixed most of tabs in test .ps1 files. 
Feel free to fix remains in new PR.